### ISSUE TITLE
HTML Report mixes tested files from all projects as if theyre all in single project

### ISSUE THREAD

**Issue by CzBuCHi** [stryker-mutator/stryker-net#2531]
**TL/DR**
Property names on "files" object in report json should be relative to solution / current directory instead of tested project directory.

Note: `Tests` table already uses parts relative to solution.

**Describe the bug**
When running stryker in solution directory with `-r html -o` arguments resulting html report show table with files from all tested projects as is they where part of single project.

Example of solution (each project has its own test project):
- ProjectA.csproj 
  - FileA1.cs
  - Dir\FileA2.cs
- ProjectB.csproj
  - FileB1.cs
  - Dir\FileB2.cs

will generate `Mutants` table like this:

| File / Directory | Mutation score,  ... |
|--- | --- |
| Dir | ... |
| FileA1.cs | ... |
| FileB1.cs | ... |

and inside 'Dir' folder:

| File / Directory | Mutation score, ... |
|--- | --- |
|FileA2.cs | ... |
|FileB2.cs | ... |

**Expected behavior**
generated table would have this structure:

| File / Directory | Mutation score, ... |
|--- | --- | 
| ProjectA | ... |
| ProjectB | ... |

and inside 'ProjectA' folder: 

| File / Directory | Mutation score, ... |
|--- | --- |
| Dir | ... |
|FileA1.cs | ... |

**Desktop (please complete the following information):**
- OS: Windows 11
- Type of project: class library
- Framework Version net48
- Stryker Version 3.8.0

**Additional context**

 i think if i defined `FileA1.cs` inside `ProjectB` report would show `ProjectA\FileA1.cs` or `ProjectB\FileA1.cs` but not both (untested).

**Comment by rouke-broersma** [comment]
Are they separated by a folder or are all project files in the same folder even if they belong to different projects? Because the report does not deal with 'projects', it only deals with real filesystem folders.

**Comment by rouke-broersma** [comment]
Mhm that's definitely a bug, we should be saving the files as absolute paths

**Comment by dupdob** [comment]
@CzBuCHi : since you have tried solution mode, could you please spend of a few minutes to weight on this question: #2543 ?

**Comment by rouke-broersma** [comment]
This should be fixed as of 3.9.2
